test/run

```diff
@@ -91,6 +91,11 @@ DOC
   echo "echomsg 'ftplugin-c'" > "$PLUG_FIXTURES/ftplugin-msg/ftplugin/c.vim"
   echo "echomsg 'ftplugin-java'" > "$PLUG_FIXTURES/ftplugin-msg/ftplugin/java.vim"
 
+  chmod +w "$PLUG_FIXTURES/cant-delete/autoload" || rm -rf "$PLUG_FIXTURES/cant-delete"
```

Comment:
Is this for debugging or optimization? It is simpler to always `rm -rf` this directory.

Reply:
Actually `rm -rf` fails without `chmod +w`.

```sh
> rm -rf cant-delete
rm: cant-delete/autoload/cant-delete.vim: Permission denied
rm: cant-delete/autoload: Directory not empty
rm: cant-delete: Directory not empty
> chmod +w cant-delete/autoload
> rm -rf cant-delete
>
```